src/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  display: flex;
```

Comment:
Don't use `display:flex` for whole body.
Make some container-wrapper and give display: flex to it

Reply:
why using display:flex for body is mistake?

Reply:
Because if you need to add another block to the body, flex applies to it too